content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -23,6 +23,79 @@ Spring Boot starter, see
 
 ## Configuration
 
+### OpenTelemetry BOM
+
+Import the `opentelemetry-bom` and `opentelemetry-instrumentation-bom-alpha`
+BOMs before using the OpenTelemetry starter.
```

Comment:
```suggestion
Import the `opentelemetry-bom` and `opentelemetry-instrumentation-bom-alpha` Bill of Materials (BOMs) before using the OpenTelemetry Starter.
```

Can you provide some context to the reader why they may want to, should or have to include this?

Reply:
I have added explanations: 3080196cec7cd23e209ac07a732209cb7e6b7eca

Reply:
I will take another look, thx